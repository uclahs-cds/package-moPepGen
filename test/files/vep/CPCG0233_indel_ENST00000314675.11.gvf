##fileformat=VCFv4.2
##mopepgen_version=0.0.1
##parser=parseVEP
##reference_index=/data/Index/GRCh38-EBI-GENCODE34
##genome_fasta=
##annotation_gtf=
##source=gindel
##CHROM=<Description='Gene ID'>
##INFO=<ID=TRANSCRIPT_ID,Number=1,Type=String,Description="Transcript ID">
##INFO=<ID=GENE_SYMBOL,Number=1,Type=String,Description="Gene Symbol">
##INFO=<ID=GENOMIC_POSITION,Number=1,Type=String,Description="Genomic Position">
#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
ENSG00000158062.20	35958	INDEL-35958-TGTCCCGGTCCCGGTCCCGGCCCCAGTCC-T	TGTCCCGGTCCCGGTCCCGGCCCCAGTCC	T	.	.	TRANSCRIPT_ID=ENST00000314675.11;GENOMIC_POSITION=chr1:26282378-26282405;GENE_SYMBOL=UBXN11
ENSG00000158062.20	35971	INDEL-35971-GTCCCGG-G	GTCCCGG	G	.	.	TRANSCRIPT_ID=ENST00000314675.11;GENOMIC_POSITION=chr1:26282387-26282392;GENE_SYMBOL=UBXN11
ENSG00000158062.20	35987	INDEL-35987-CGG-C	CGG	C	.	.	TRANSCRIPT_ID=ENST00000314675.11;GENOMIC_POSITION=chr1:26282375-26282376;GENE_SYMBOL=UBXN11
ENSG00000158062.20	35987	INDEL-35987-CGGTCCCGG-C	CGGTCCCGG	C	.	.	TRANSCRIPT_ID=ENST00000314675.11;GENOMIC_POSITION=chr1:26282369-26282376;GENE_SYMBOL=UBXN11
ENSG00000158062.20	35996	INDEL-35996-TCCCGGCCCCA-T	TCCCGGCCCCA	T	.	.	TRANSCRIPT_ID=ENST00000314675.11;GENOMIC_POSITION=chr1:26282358-26282367;GENE_SYMBOL=UBXN11
