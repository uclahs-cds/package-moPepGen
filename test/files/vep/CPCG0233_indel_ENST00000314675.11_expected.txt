APKSSLKFSPGPCPGPGPGPSPSR
APKSSLKFSPGPCPGPGPSPSR
APKSSLKFSPGPSVPVSR
CPGPGPGPSPCPGPSPSPQ
FSPGPCPGPGPGPSPSR
FSPGPCPGPGPGPSPSRPQSR
FSPGPCPGPGPGPSPSRPQSRSR
FSPGPCPGPGPGPSPSRSRPQSR
FSPGPCPGPGPGPSPSRSRPQSRSR
FSPGPCPGPGPSPSR
FSPGPCPGPGPSPSRPQSR
FSPGPCPGPGPSPSRPQSRSR
FSPGPCPGPGPSPSRSRPQSR
FSPGPCPGPGPSPSRSRPQSRSR
FSPGPSVPVSR
FSPGPSVPVSRSR
SRPQSLSWTQSQPP
SRPQSRSR
SRPQSRSRSRPQSLSWTQSQPP
SRSRPQSLSWTQSQPP
SSLKFSPGPCPGPGPGPSPSR
SSLKFSPGPCPGPGPGPSPSRPQSR
SSLKFSPGPCPGPGPSPSR
SSLKFSPGPCPGPGPSPSRPQSR
SSLKFSPGPCPGPGPSPSRPQSRSR
SSLKFSPGPCPGPGPSPSRSRPQSR
SSLKFSPGPSVPVSR
SSLKFSPGPSVPVSRSR
